neptune-gremlin.js

```diff
@@ -104,6 +120,29 @@ class Connection {
 
     }
 
+    /**
+     * Get the graph traversal, which might be using a partition strategy.
+     * 
+     * @returns 
+     */
+    getG() {
+        console.log("About to get traversal")
+        let g = traversal().withRemote(this.connection)
+        let gx = g
```

Comment:
Rather than have this mutating variable, just return early in the `if` statement.

```js
        if (this.partition) {
            // ...
            return g.withStrategies(strategy)
        }
```

Reply:
Done